log/static/static.go

```diff
@@ -0,0 +1,62 @@
+// Package static provides a simple static logger implementation for easy migration of existing pre-v2 code.
+//
+// The package is usually imported aliased as "log" to be compatible with the v1 approach.
+//
+// The static logger instance needs to be initialized during startup of the application using SetLogger.
+package static
+
+import (
+	"sync"
+
+	"github.com/ViaQ/logerr/v2/log"
+	"github.com/go-logr/logr"
+)
+
+var (
+	lock   sync.RWMutex
+	logger = log.NewLogger("uninitialized")
```

Comment:
One idea that came out of a question on the explore-team channel: Should we have a high verbosity for the "uninitialized logger", so that usage of the uninitialized logger is more visible? I would assume that this will make the current unit-test code quite chatty though.

Reply:
I would say rather no as you point out it makes our dev side chatty w/o gaining any benefits.